app/helpers/document_list_helper.rb

```diff
@@ -0,0 +1,7 @@
+module DocumentListHelper
+  def convert_string_to_date_object(updated)
+    unless updated.is_a?(ActiveSupport::TimeWithZone)
+      updated = ActiveSupport::TimeZone['UTC'].parse(updated)
```

Comment:
I think this should be `Time.zone.parse. This will use the app’s timezone rather than UTC (eg London): https://github.com/alphagov/government-frontend/blame/master/app/presenters/updatable.rb#L36

More context in this commit:
https://github.com/alphagov/government-frontend/commit/737211f87e9a9dbf4bcbb3b518a91fec317df50a